src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -121,6 +121,11 @@ class ApplicationImpl : public virtual Application,
   }
   void set_is_navi(bool allow);
 
+  inline bool is_rc() const {
+    return is_rc_;
+  }
+  void set_is_rc(bool allow);
```

Comment:
@BSolonenko 
Please add an empty line between functions

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2048/commits/3ffd281451b051e9ead29c3a877a6678109321ac